src/main/scala/servers/Metals.scala

```diff
@@ -54,18 +54,12 @@ object Metals extends ScalaLanguageServer { server =>
     "--main", "scala.meta.metals.Main"
   )
 
-  val commands = Seq(
-    "build-import",
-    "build-connect",
-    "sources-scan",
-    "doctor-run",
-  ).map { cmd =>
-    cmd -> { activeServer: ActiveServer => _: js.Any =>
-      activeServer.connection.executeCommand(
-        new ExecuteCommandParams(command = cmd)
-      )
-    }
-  }.toMap
+  val commands = Map(
+    "build-import" -> "Import build",
+    "build-connect" -> "Connect to build server",
+    "sources-scan" -> "Rescan sources",
+    "doctor-run" -> "Run Doctor",
```

Comment:
Let's do "Run doctor" for consistent capitalization, I'll update the vscode extension

Reply:
Here, it's by accident, but it's capitalized per word in another place anyway. I did this for consistency with the rest of the Atom command palette

Reply:
<img width="326" alt="screenshot 2018-12-02 at 18 18 41" src="https://user-images.githubusercontent.com/766656/49342697-bc38d500-f65e-11e8-9243-9a71444a944a.png">

See other commands